app/components/footer.module.css

```
@@ -0,0 +1,14 @@
+.footer {
+  margin-top: 60px;
+  background: linear-gradient(150deg, #0b69eb 70%, #ffffff 100%);
+  padding: 20px;
```

Stylelint (17.10.0)

[error] Duplicate property "padding" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)